docs/advanced/polygon-testnet-on-metamask.md

```diff
@@ -0,0 +1,31 @@
+---
+id: polygon-testnet-on-metamask
+title: Setup Polygon Testnet on Metamask
+sidebar_label: Setup Polygon Testnet on Metamask
+---
+
+> _NOTE: Please make sure you have already installed Metamask!_
+
+Please follow the steps to add Polygon's Mumbai testnet:
+
+1. Navigate to [alchemy.com](https://www.alchemy.com/)
```

Comment:
do you actually need an alchemy account for the polygon mumbai rpc url?

Reply:
ya, I tried that method from polygonscan, the endorsement chain wont load. and there are other ways, but this way we can guarantee we will have a working rpc url everytime.

Reply:
y wont endorsement chain load?

Reply:
from what i see it will add into metamask. but this rpc url is the free one, it can only access latest 1000 block, thus the error of endorsement chain happens. cos the endorsement chain need to access the very 1st block.

Reply:
adding the need of registering an alchemy account prior to adding another network introduces another layer of friction with users imo. if there is a problem with endorsement chain, should that be looked into or fixed or workaround?

Reply:
understand that @simboonlong, its not a problem with endorsement chain but the problem with connecting into the polygon network using the rpc url.

Reply:
1) can list down the rpc url that fails the endorsement chain
2) plus the document that results in the failed endorsement chain?

how did you come to this conclusion for "it can only access latest 1000 block"?

Reply:
erm @puayhiang for the subgraph, i dont think it is related at the moment. currently it seems that the metamask can't access the network given the rpc url. 

@cavacado it is in the error message, but need to dig deeper a bit. 
@cavacado for your point 1, is it u suggest we add a list of failed rpc url in the docs?